protoc_plugin/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 17.0.5
+
+* Generate specialized getters for String, int, and bool with usual
+  default values.
```

Comment:
Add that the minimum supported protobuf version has gone up.
I guess technically that turns this into a breaking change of protoc_plugin.